src/audio-worklet/free-queue/examples/hello-webassembly/index.njk

```diff
@@ -0,0 +1,33 @@
+---
+eleventyNavigation:
+  key: hello-webassembly
+  title: Using FreeQueue with WebAssembly
+  parent: audio-worklet
+to_root_dir: ../../../../
+---
+
+{% extends to_root_dir + "_includes/base.njk" %}
+
+{% block content %}
+
+<script src="./coi-serviceworker.js"></script>
```

Comment:
At L12:
`{# Inject the service worker to workaround COOP/COEP restriction on SharedArrayBuffer. See https://github.com/gzuidhof/coi-serviceworker for more details. #}`